cloud-energy-collection/iriscasttools/main.py

```diff
@@ -0,0 +1,27 @@
+"""
+Collects energy usage metrics using IPMI, as well as other metrics such as CPU and RAM usage.
+"""
+import utils
+
+
+def get_iriscast_stats(csv=False, include_header=False):
```

Comment:
This should be wrapped in a `if __main__` too:
https://stackoverflow.com/questions/419163/what-does-if-name-main-do

Like:
```
if __main__:
   get_iriscast_stats(...)
```